added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros

huiguoo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#49121 added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros**
* #49040 converted current debugging statements in LLVM codegen to jit-logging statements #48771



Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

huiguoo committed:
added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros

[ghstack-poisoned]

huiguoo commented on torch/csrc/jit/tensorexpr/llvm_codegen.cpp:
```diff
@@ -519,7 +519,7 @@ void LLVMCodeGenImpl::emitKernel(
   llvm::SmallVector<char, 0> asmBuffer;
   llvm::raw_svector_ostream asmStream(asmBuffer);
-  if (is_enabled(__FILE__, ::torch::jit::JitLoggingLevels::GRAPH_DEBUG)) {
+  if GRAPH_DEBUG_ENABLED {
```
Clang format suggests to split `if` and `GRAPH_DEBUG_ENABLED {` in 2 lines. Below is the diff,
```
-  if GRAPH_DEBUG_ENABLED {
-    module_->print(asmStream, nullptr);
-  }
+  if
+    GRAPH_DEBUG_ENABLED {
+      module_->print(asmStream, nullptr);
+    }
```
Looks weird to me - add it or drop it?

ZolotukhinM replied:
Maybe put it in parenthesis?

ZolotukhinM reviewed (approved):
Looks good to me!

huiguoo committed:
Update on "added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros"


Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

[ghstack-poisoned]

huiguoo committed:
Update on "added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros"


Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

[ghstack-poisoned]

huiguoo committed:
Update on "added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros"




Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

[ghstack-poisoned]

huiguoo committed:
Update on "added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros"




Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

[ghstack-poisoned]

huiguoo committed:
Update on "added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros"




Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

[ghstack-poisoned]

huiguoo committed:
Update on "added macros in jit logging to check whether loggings are enabled; replaced similar checks in LLVM codegen with such macros"




Differential Revision: [D25445971](https://our.internmc.facebook.com/intern/diff/D25445971)

[ghstack-poisoned]
